Formula/m/mariadb.rb

```diff
@@ -1,10 +1,9 @@
 class Mariadb < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-11.0.2/source/mariadb-11.0.2.tar.gz"
-  sha256 "3c71576cad0ea416882270e383f94cc89693b7fbcce1fa4f306fe3e931e464ae"
+  url "https://downloads.mariadb.com/MariaDB/mariadb-11.0.3/source/mariadb-11.0.3.tar.gz"
```

Comment:
```suggestion
  url "https://archive.mariadb.org/mariadb-11.0.3/source/mariadb-11.0.3.tar.gz"
```

Reply:
Will do after the fix is finalised. (11.1.2 is also affected by the `libfmt` issue.)

Same for the remaining `mariadb` PRs.

Reply:
Applied suggestion (and updated to 11.1.2).